htsinfer/infer_read_orientation.py

```diff
@@ -22,5 +21,45 @@ def infer(
         LIBTYPE string according to Salmon documentation, cf.
         https://salmon.readthedocs.io/en/latest/library_type.html
     """
+    #create .tmp directory in "inputDir" and retuns .tmp path
+    tmpDir = mkTmpDir(inputDir)
+    
+    # call functions of issues #25, 26, 27 and 34
+    
+    # delete .tmp
+    rmTmpDir(tmpDir)
+        
     # implement logic
     return "U"
+    
+
+def mkTmpDir(inputDir):
```

Comment:
It is a convention to use "snake case" for function names and variables, and "camel case" for classes. So in this case, as you are writing a function, the name of the function should be `mk_tmp_dir(input_dir)` (and similarly for the other function below). See [Python's style guidelines](https://www.python.org/dev/peps/pep-0008/) for more details